app/views/pregnancies/_pledge.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="modal pledge_modal">
+	<div class="pledge_modal_header">
+		<span class="pledge_header_text">Create Pledge</span>
+		<span class="pledge_close glyphicon glyphicon-remove close" data-dismiss="modal" aria-label="Close"></span>
+	</div>
+
+	<div class="pledge_modal_screen1">
+		<div class="pledge_intro">Confirm the following information is correct:</div>
+		<div class="pledge_form">
+			<div><span class="pledge_modal_category">Patient name:</span> <%= @pregnancy.patient.name %></div>
+			<div><span class="pledge_modal_category">Patient ID:</span> <%= @pregnancy.patient.primary_phone %></div>
+			<div><span class="pledge_modal_category">Pledge amount:</span> $100</div>
+			<div><span class="pledge_modal_category">Appointment date:</span> <%= @pregnancy.appointment_date %></div>
+			<div><span class="pledge_modal_category">Clinic name:</span>Name</div>
+			<div><span class="pledge_modal_category">Clinic location:</span>City, State</div>
+		</div>
+		<a class="pledge_process_btn pledge_cancel"  data-dismiss="modal" aria-label="Close">Cancel</a>
+	 	<a class="btn btn-primary btn-sm pledge_process_btn pledge_submit">Continue</a>
+	</div>
+
+	<div class="pledge_modal_screen2">
+		<div class="pledge_intro">Review this preview of your pledge:</div>
+			<a class="pledge_process_btn pledge_back_submit"><span class="glyphicon glyphicon-chevron-left"></span>Go back</a>
+			<a class="btn btn-primary btn-sm pledge_process_btn pledge_continue">Download and continue</a>		
+	</div>
+
+	<div class="pledge_modal_screen3">
+		<div class="pledge_intro">
+			<p>Awesome, you generated a DCAF Pledge! Thanks!</p>
+			<p>Your pledge should appear in your computer's downloads. Find the document and go to <a href="http://www.onlinefaxes.com">onlinefaxes.com</a> to submit the pledge for your client.</p>
```

Comment:
Can we add a `target='_blank'` to this link, so that it opens up in a new tab or window?